lib/shared/config.rb

```diff
@@ -0,0 +1,48 @@
+require 'json'
+
+require_relative 'errors/file_not_found'
+
+module GoogleDrive
+  class Config
+    attr_accessor :client_id, :client_secret, :scope, :redirect_uris, :access_token
+
+    def initialize
+      return unless valid?
```

Comment:
I prefer to omit this check, and just let load_config_file throw a standard SystemCallError when the file is not found, rather than defining and throwing our own exception class.